src/config/dynamic-theme-fixes.config

```diff
@@ -3914,6 +3926,15 @@ img.mega-menu-logo
 
 ================================
 
+cmt.com
```

Comment:
Ditto.

Reply:
@Gusted, echoing what @q1800 said, this is the actual, non-redirected website (at least within the US).